langgraph/channels/dynamic_barrier_value.py

```diff
@@ -60,11 +60,6 @@ def from_checkpoint(
             pass
 
     def update(self, values: Sequence[Union[Value, WaitForNames]]) -> None:
-        # switch to "priming" state after reading
```

Comment:
Dynamic barrier value only used for the `then` clause of conditional edges ya ? (I also see it used in the Send API though - guess the WaitForNames is used to block undefined send and mid-conditional edge logic or something? cant tell at a glance).  So 'priming' is its natural state and then collects nodes as the branches are seen

Reply:
Yea that's how dynamic barrier works, it's triggered in two stages, first send list of names to wait for, then wait for those names